### DISCUSSION TITLE
Header zoom from any line instead of requiring cursor to be on the Header line itself

### DISCUSSION THREAD

**Discussion by GitMurf** [vslinko/obsidian-zoom#8] [Ideas]
Currently for header zooming you have to have your cursor on the header line itself. Is there a way that we can have it zoom from any line? So if I am on a regular line underneath a header and do Ctrl + "." then it zooms into that header that I am currently under? Instead of having to move my cursor to the header line itself? Thanks!

Yes. So lets say I have this:

```
ln1 # Some header
ln2 
ln3 ## Another header
ln4 
ln5 Typing some stuff
ln6 Typing more stuff and want to zoom from here by pressing ctrl + "." {CURSOR HERE}
```

And then I want it to just zoom into the "## Another header" so it is in focus and I can continue typing from line 6 where my cursor was.

**Comment by Creling** [comment]
Yes, I want this feature as well. Currently, when I review my notes and want to add some paragraphs, I won't be able to zoom in until I scroll the page up and move my cursor to the heading line.

I think it will be of great convenience if we could zoom in from any line under a heading.

**Comment by zcysxy** [comment]
I want this feature too!

**Comment by vslinko** [comment]
Thank you all for the feedback. This will be implemented in the future.
